Neos.Cache/Classes/Backend/SimpleFileBackend.php

```diff
@@ -484,7 +547,9 @@ protected function configureCacheDirectory()
      */
     protected function readCacheFile(string $cacheEntryPathAndFilename, int $offset = null, int $maxlen = null)
     {
-        for ($i = 0; $i < 3; $i++) {
+        $tries = 10;
+        $waitTime = $this->maxAcceptedReadLatencyMicroSeconds / $tries;
+        for ($i = 0; $i < $tries; $i++) {
```

Comment:
```suggestion
        // We do a fixed amount of retries with a fixed sleep because reads do not block each other out unlike writes
        for ($i = 0; $i < $tries; $i++) {
```